mod/profile/views/default/profile/css.php

```diff
@@ -59,12 +59,7 @@
 #profile-details .wire-status {
 	margin-top: 10px;
 }
-#profile-details .odd {
-	border-bottom: 1px solid #DCDCDC;
-	margin: 0;
-	padding: 5px 0;
-}
-#profile-details .even {
+.profile-fields > div {
```

Comment:
added a issue for reworking the profile field listings on profile/group https://github.com/Elgg/Elgg/issues/10412 so we can keep it separate from this PR